server/pages/index.js

```diff
@@ -278,7 +278,7 @@ function setUpLoggedInRoute( req, res, next ) {
 		start = new Date().getTime();
 
 		debug( 'Issuing API call to fetch user object' );
-		user( req.get( 'Cookie' ), function( error, data ) {
+		user( `wordpress_logged_in=${ req.cookies.wordpress_logged_in }`, function( error, data ) {
```

Comment:
We could simplify the signature of `user()` by passing `req.cookies.wordpress_logged_in` directly. This would also make the code of [`user-bootstrap`](https://github.com/Automattic/wp-calypso/blob/53bde8c3f9c4668e62dbd2c670aaacc2bd820427/server/user-bootstrap/index.js#L20-L30) much simpler:

```javascript
module.exports = function( authCookieValue, callback ) {
  // create HTTP Request object
  var req = superagent.get( url ),
    hmac, cookies, hash;

  hmac = crypto.createHmac( 'md5', API_KEY );
  hmac.update( authCookieValue );
  hash = hmac.digest( 'hex' );
  
  req.set( 'Authorization', 'X-WPCALYPSO ' + hash );
  req.set( 'Cookie', `wordpress_logged_in=${ authCookieValue }` );
  req.set( 'User-Agent', 'WordPress.com Calypso' );

  // start the request
  [...]
```